assets/src/edit-story/components/canvas/useCanvasKeys.js

```diff
@@ -148,6 +152,26 @@ function useCanvasKeys(ref) {
     [arrangeSelection]
   );
 
+  // Edit mode
+  useGlobalKeyDownEffect(
+    'enter',
+    () => {
+      if (selectedElements.length === 0) {
+        return;
+      }
+
+      const { type, id } = selectedElements[0];
+      const { hasEditMode } = getDefinitionForType(type);
+      // Only handle Enter key for editable elements
+      if (!hasEditMode) {
+        return;
+      }
+
+      setEditingElement(id);
```

Comment:
Does this conflict with the text editor, which provides a lot more data when setting the editing element? Alternatively, we can let each frame type to support the <kbd>Enter</kbd> invidually.

Also, this needs a Karma test. There some related tests [here](https://github.com/google/web-stories-wp/blob/master/assets/src/edit-story/components/canvas/karma/keys.karma.js).

Reply:
The text editor has more specificity for the <kbd>Enter</kbd> event so it should be able to take over. I'll add tests!